packages/nimble-components/src/table/template.ts

```diff
@@ -26,13 +26,14 @@ const isTableColumn = (): ElementsFilter => {
 
 // prettier-ignore
 export const template = html<Table>`
-    <template role="table">
+    <template role="table" ${children({ property: 'columns', filter: isTableColumn() })}>
         <div class="table-container">
             <div role="rowgroup" class="header-container" style="margin-right: ${x => x.virtualizer.headerContainerMarginRight}px;">
                 <div class="header-row" role="row">
                     ${repeat(x => x.columns, html<TableColumn>`
-                        <${DesignSystem.tagFor(TableHeader)} class="header">
-                            ${x => x.textContent}
+                        <${DesignSystem.tagFor(TableHeader)} 
```

Comment:
nit: It looks like the only thing that changed with this line is wrapping part of it to the next line. Since it isn't that long of a line, can this change be reverted?